src/app/api/room/route.ts

```diff
@@ -0,0 +1,83 @@
+import { randomBytes } from "crypto";
+
+function generateTrulyRandom6DigitNumber(): number {
+  const buffer = randomBytes(6);
+
+  const number = buffer.readUInt32BE(0);
+
+  return number + 1;
+}
+
+class Huddle01CreateRoomError extends Error {
+  status: number;
+  constructor(message: string, status: number) {
+    super(message);
+    this.status = status;
+  }
+}
+
+export async function GET(request: Request) {
+  const API_KEY = process.env.HUDDLE_API_KEY;
+  const id = generateTrulyRandom6DigitNumber();
+
+  if (!API_KEY) {
+    const jsonResponse = new Response(
+      JSON.stringify({ msg: "Missing API KEY" }),
+      {
+        status: 404,
+        statusText: "API Key not found",
+      }
+    );
+
+    return jsonResponse;
+  }
+  try {
+    const response = await fetch(
+      "https://api.huddle01.com/api/v1/create-iframe-room",
+      {
+        method: "POST",
+        body: JSON.stringify({
+          title: `Huddle01-Test${id}`,
```

Comment:
Don't need to use random digit generator for room title, simply pass `Huddle01 iFrame Room`.